lib/chef/provider/script.rb

```diff
@@ -18,6 +18,7 @@
 
 require "tempfile"
 require "chef/provider/execute"
+require "chef/win32/security" if Chef::Platform.windows?
```

Comment:
yeah, because that's outside the class. If you move the require into `grant_alternate_user_read_access` and check `node[:platform_family] == windows` that'll work.

While I understand what @lamont-granquist is getting at, we've got to go around and fix all of this soon for https://github.com/chef/chef/issues/5771 anyway.